src/main.tsx

```diff
@@ -1,4 +1,4 @@
-import './installSesLockdown';
+await import('./installSesLockdown');
```

Comment:
I like the promise approach but it's surprising to await the import.

How about a named export instead, like `lockedDown` and here we can have,
```suggestion
import {lockedDownWithKeplr} from './installSesLockdown';
```
Then `await lockedDown;` before createRoot.

Reply:
Thanks for pointing out that this suggestion would allow the other imports to possibly happen before lockdown.

While awaiting the import is "odd" it expresses the requirement: nothing else should import until the promise completes.  👌